client/css/style.css

```diff
@@ -1614,8 +1624,8 @@ kbd {
 	animation-fill-mode: forwards;
 	-webkit-animation-timing-function: ease-in;
 	animation-timing-function: ease-in;
-	-webkit-animation-delay: .4s;
-	animation-delay: .4s;
+	-webkit-animation-delay: .1s;
+	animation-delay: .1s;
```

Comment:
As I mentioned before, it was done to match new transition effects. There are cases when these 2 animations happen simultaneously (settings, help, toggle user list, etc) , and having different animation duration makes them look bad.